nengo_interfaces/airsim.py

```diff
@@ -25,76 +35,257 @@ class AirSim(nengo.Process):
     dt : float, optional (Default: 0.01)
         The time step for the simulation dynamics.
     run_async : boolean, optional (Default: False)
-        If true, run AirSim simulation independent of Nengo simulation. If false,
-        run the two in lockstep, i.e., one Nengo time step then one AirSim time step
-    render_params : dict, optional (Default: None)
-        'cameras' : list
-        'resolution' : list
-        'update_frequency' : int, optional (Default: 1)
-        'plot_frequency' : int, optional (Default: None)
-    track_input : bool, option (Default: False)
-        If True, store the input that is passed in to the Node in self.input_track.
-    input_bias : int, optional (Default: 0)
-        Added to the input signal to the Mujoco environment, after scaling.
-    input_scale : int, optional (Default: 1)
-        Multiplies the input signal to the Mujoco environment, before biasing.
+        If true, run AirSim simulation independent of Nengo simulation.
+        If false, run the two in lockstep, i.e., one Nengo time step then one
+        AirSim time step
+    camera_params : dict, optional (Default: None)
+        'camera_name' : string or int
+        'fps' : float
+        'save_name' : string
```

Comment:
Might be useful to support setting this to `None` to have the option of not saving images. Currently if you set it to `None` it will run fine without saving them, but it will print out a bunch of error messages.

Reply:
Good catch, I've made the changes to avoid the error. We'll have to make a PR later to get the image information sent back out of the node. At the moment it was just meant for saving images and sending drone feedback, but there will definitely be cases where we need vision feedback as well.